import grpc
import peerCommunications_pb2
import peerCommunications_pb2_grpc

class Client:

    def __init__(self):
        pass

    def QueryFileList(self, request, context):
        pass

    def UploadFile(self, request, context):
        pass

    def DownloadFile(self, request, context):
        pass

    def Ping(self, request, context):
        pass
